WordPress/wordpress-playground
# Blueprints: Add a "description" field

@tellyworth noted in #784 that a description property would be handy for developers to describe the purpose of a blueprint. This commit introduces it.

Testing instructions

1. Confirm the TypeScript type is generated and the Blueprint schema is updated.
2. Confirm the tests pass
3. There is no action to take since this new fields doesn't do anything

Closes #784
